### ISSUE TITLE
Moonreader V9.3 seems to be incompatible with 2.7.1 (2.7.2?)

### ISSUE THREAD

**Issue by erdoking** [mikespub-org/seblucas-cops#85]
After updating the docker container linuxserver/cops to v2.7.2 MoonReader+ are unable to open librarys. Librarys are listed but without function - maybe with wrong url because of loop.
Downgrade to 2.6.1 fixed the issue.

```
x.x.x.x - - [29/May/2024:10:22:34 +0200] "GET /?db=0 HTTP/1.1" 302 - "-" "OPDS/Stanza iPhone/Aldiko/Moon+ Reader(Android)"
x.x.x.x - - [29/May/2024:10:22:34 +0200] "GET /feed.php HTTP/1.1" 200 1962 "-" "OPDS/Stanza iPhone/Aldiko/Moon+ Reader(Android)"
x.x.x.x - - [29/May/2024:10:22:36 +0200] "GET /?db=0 HTTP/1.1" 302 - "-" "OPDS/Stanza iPhone/Aldiko/Moon+ Reader(Android)"
x.x.x.x - - [29/May/2024:10:22:36 +0200] "GET /feed.php HTTP/1.1" 200 1962 "-" "OPDS/Stanza iPhone/Aldiko/Moon+ Reader(Android)"
```

**Comment by mikespub** [comment]
Hi @erdoking

I'm not sure I understand what you mean by "listed without function" here. Can you show an example of what the /feed.php output is then?

**Comment by shaoyangx** [comment]
I also encountered the same problem. 
In version 2.7.2, when there are multiple book libraries 
$config['calibre_directory'] = array ("ebook" => "/books/ebook/", "资料" => "/books/资料/", "测试" => "/books/测试/");
 opds connection and clicking the book library will loop infinitely on the book library selection page and cannot enter the book library. 
However, when the configuration file is a single book library, it can work normally. 
$config['calibre_directory'] = '/books/测试/';
and in the feed.php file, the only difference is whether there is /feed.php in the <link href="/feed.php?db=0" connection. 
Judging from the output, the only difference in the output after entering the library is the version number.
After switching back to 2.6.1, it returned to normal.
The configuration files are exactly the same, no changes were made
[feed.zip](https://github.com/user-attachments/files/15685715/feed.zip)

**Comment by mikespub** [comment]
Thanks @shaoyangx that helps clarify the problem with the 2.7.x changes - sorry about that...

**Comment by mikespub** [comment]
Included in [release 2.7.3](https://github.com/mikespub-org/seblucas-cops/releases/tag/2.7.3)
